server/src/main/java/com/cloud/resource/ResourceManagerImpl.java

```diff
@@ -2481,7 +2482,7 @@ public void deleteRoutingHost(final HostVO host, final boolean isForced, final b
                 try {
                     _vmMgr.destroy(vm.getUuid(), false);
                 } catch (final Exception e) {
-                    final String errorMsg = "There was an error Destory the vm: " + vm + " as a part of hostDelete id=" + host.getId();
+                    String errorMsg = String.format("There was an error when destroying the VM: %s as a part of hostDelete: %s", vm, host);
```

Comment:
Could you to take a look into the VM's `toString` implementation?

https://github.com/apache/cloudstack/blob/7835c0812062381c7e5dd5ab9d719297cd3187dc/engine/schema/src/main/java/com/cloud/vm/VMInstanceVO.java#L508-L510

Seems like there will be duplicated words:
`There was an error when destroying the VM: VM Instance...`

It would apply to every time we log the VM.